static/js/stream_edit.js

```diff
@@ -634,7 +634,7 @@ exports.initialize = function () {
     $("#subscriptions_table").on("click", ".sub_unsub_button", function (e) {
         const sub = get_sub_for_target(e.target);
         const stream_row = $(this).parent();
-        subs.sub_or_unsub(sub);
+        subs.sub_or_unsub(sub, this);
```

Comment:
We generally don't like using `this` when we had the event as `e` in a click handler; can you work with something based on `e.target` or `e.currentTarget` instead?